### DISCUSSION TITLE
YouTube frontend alternative

### DISCUSSION THREAD

**Discussion by AdmiralAwesome** [tobychui/zoraxy#73] [Ideas]
Since YouTube is pushing down on Ads and adblockers, maybe a script like Piped? Or Invidious? Or something else that I'm not aware of.

**Comment by tobychui** [comment]
I feel like this is more like a [AdGuard DNS](https://adguard-dns.io/en/blog/adguard-dns-2-0-goes-open-source.html) (or similar system's) jobs than the reverse proxy job 🤔

**Reply by AdmiralAwesome** [reply]
Some apps (like YouTube on iOS) use their own, embedded in app code nameservers so AdGuard/pihole will not block ads inside them.

**Reply by tobychui** [reply]
I am not sure how others in-app ads works, but I am 100% sure this is not a reverse proxy's job to hide them for you. You might take a look at other VPN / hardware firewall projects which might be a bit closer to what you are looking for.
